src/App.tsx

```diff
@@ -1,18 +1,33 @@
-import React from 'react';
+import { ChangeEvent, useState } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const countItems = 42;
+
+export const App = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [countItemsPerPage, setCountItemsPerPage] = useState(3);
+
+  const handlerChangeCountPerPage = (event: ChangeEvent<HTMLSelectElement>) => {
+    setCountItemsPerPage(+event.target.value);
+    setCurrentPage(1);
+  };
+
+  const fromPage = 1 + (currentPage - 1) * countItemsPerPage;
```

Comment:
```suggestion
  const fromItem = 1 + (currentPage - 1) * countItemsPerPage;
```
mb let's rename it this way 🤔